app/countwords/views.py

```diff
@@ -0,0 +1,53 @@
+import os
+import re
+import tempfile
+
+import docx
+import requests
+from flask import Blueprint, request, current_app
+from flask.helpers import make_response
+from flask.json import jsonify
+
+countwordsBlueprint = Blueprint('countwords', __name__)
+
+
+@countwordsBlueprint.route('/countwords', methods=["GET"])
+def countwords():
+    f_uid = request.args.get('f', type=str)
+    if f_uid is None:
+        return make_response("You must send files", 400)
+    result = []
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        doc = fetchFile(f_uid, tmp_dir)
+        for p in doc.paragraphs:
+            w_map = countWords(p)
+            if len(w_map) == 0:
+                continue
+            result.append(w_map)
+
+    return make_response(jsonify(result), 200)
+
+
+def fetchFile(uid, temp_dir):
+    url = current_app.config['LINKER_URL'] + uid
+    r = requests.get(url, allow_redirects=True)
```

Comment:
For uniformity, I'd rather have it like [here](https://github.com/Bnei-Baruch/archive-unzip/blob/master/app/doc2html/views.py#L144)
Moreover, better check with mdb that this file exists and it's indeed of a supported type before you fetch it.
Maybe try to find a way to extract this functionality to some place common and use it everywhere.

Reply:
@edoshor 
I add common class [fetchFile](https://github.com/Bnei-Baruch/archive-unzip/blob/compare_files/app/fetchfile.py)
But I can't find easy way replace [this code](https://github.com/Bnei-Baruch/archive-unzip/blob/fdf447c431a68c18c7c4c659891c335a46ae9943/app/doc2html/views.py#L102)
If I will change it can be bugs on the checked place, not sure if need to do it. 
what you say?

Reply:
ok. don't change the other place.